packages/copyable/src/Copyable.spec.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import ClipboardJS from 'clipboard';
+import {
+  render,
+  fireEvent,
+  waitFor,
+  waitForElementToBeRemoved,
+} from '@testing-library/react';
+import { Context, jest } from '@leafygreen-ui/testing-lib';
+import Copyable from '.';
+
+describe('packages/copyable', () => {
+  test('text elements are rendered', () => {
+    const { getByText } = render(
+      <Copyable label="Label" description="Description">
+        Hello world
+      </Copyable>,
+    );
+
+    expect(getByText('Label')).toBeVisible();
+    expect(getByText('Description')).toBeVisible();
+    expect(getByText('Hello world')).toBeVisible();
+  });
+
+  describe('copy button', () => {
+    test('has tooltip when clicked', async () => {
+      await Context.within(
+        jest.spyContext(ClipboardJS, 'isSupported'),
+        async spy => {
+          spy.mockReturnValue(true);
+
+          const { getByText, queryByText } = render(
+            <Copyable label="Label" description="Description">
+              Hello world
+            </Copyable>,
+          );
+
+          expect(queryByText('Copied!')).not.toBeInTheDocument();
+
+          const copyButton = getByText('Copy');
+          expect(copyButton).toBeVisible();
+
+          await Context.within(
+            jest.spyContext(ClipboardJS.prototype, 'onClick'),
```

Comment:
What does the usage of Context.within provide here? Just an easy way to cleanup the onClick spy?

Reply:
also makes the test more self-contained since the spy will be cleaned up even if the test fails